src/txn/txn_ckpt.c

```diff
@@ -1335,8 +1280,7 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
         __drop_list_execute(session, session->ckpt_drop_list);
         __wt_scr_free(session, &session->ckpt_drop_list);
     }
-    if (full || name != NULL)
-        WT_ERR(__wt_meta_sysinfo_set(session, full, name, namelen));
+    WT_ERR(__wt_meta_sysinfo_set(session, name, namelen));
```

Comment:
I mistakenly thought the `name != NULL` check was missed here, but it's actually present within `__wt_meta_sysinfo_set`